target/linux/ath79/dts/qcn5502_netgear_ex7300v2.dts

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Netgear EX7300v2";
+	compatible = "netgear,ex7300v2", "qca,qca9560";
```

Comment:
model = "Netgear EX7300 v2";
	compatible = "netgear,ex7300-v2", "qca,qca9560";

Consider adding a string for the new board between the two existing ones in compatible.

Reply:
Done